models/token.js

```diff
@@ -2,16 +2,21 @@
 
 const Joi = require('joi');
 const mutate = require('../lib/mutate');
+const TOKEN_LENGTH = 256;
```

Comment:
Can you provide extra comment above this line about why the token_length is 256 and why the HASH_LENGTH is 256 / 24?